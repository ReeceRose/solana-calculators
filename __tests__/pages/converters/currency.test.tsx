import { render } from '@testing-library/react';

import CurrencyConverter from '../../../pages/converters/currency';

test('converters/currency page matches snapshot', () => {
  const { asFragment } = render(<CurrencyConverter />);
  expect(asFragment()).toMatchSnapshot();
});
