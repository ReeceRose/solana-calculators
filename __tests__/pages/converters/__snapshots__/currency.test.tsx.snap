// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`converters/currency page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex-grow w-full px-4 mt-12 mb-0"
  >
    <div
      class="flex flex-wrap"
    >
      <div
        class="w-full px-4 mx-auto lg:w-10/12"
      >
        <div
          class="relative flex flex-col w-full min-w-0 break-words bg-gray-700 border-0 rounded-lg shadow-lg"
        >
          <div
            class="px-6 py-6 mb-0 bg-gray-900 rounded-t"
          >
            <div
              class="flex justify-between text-center"
            >
              <h6
                class="text-xl font-bold text-solana-green"
              >
                Currency Converter
              </h6>
              <button
                class="px-4 py-2 mr-1 text-xs font-bold uppercase transition-all duration-150 ease-linear bg-gray-500 rounded shadow outline-none text-solana-green active:bg-gray-600 hover:shadow-md focus:outline-none"
                type="button"
              >
                Convert
              </button>
            </div>
          </div>
          <div
            class="flex-auto px-4 lg:px-10"
          >
            <hr
              class="my-6 border-gray-50 border-b-1"
            />
            <h6
              class="text-sm font-bold uppercase text-gray-50"
            >
              Current Sol price (USD): 
              <span
                class="font-extrabold text-solana-green"
              >
                0 USD
              </span>
            </h6>
          </div>
          <div
            class="flex-auto px-4 mb-6 lg:px-10"
          >
            <form>
              <hr
                class="my-6 border-gray-50 border-b-1"
              />
              <h6
                class="mt-3 mb-6 text-sm font-bold uppercase text-gray-50"
              >
                Converter
              </h6>
              <div
                class="flex flex-wrap"
              >
                <div
                  class="w-full px-4 lg:w-6/12"
                >
                  <div
                    class="relative w-full mb-3"
                  />
                </div>
              </div>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
